packages/create-cloudflare/src/helpers/interactive.ts

```diff
@@ -221,6 +221,19 @@ export const confirmInput = async (opts: ConfirmOptions) => {
 };
 
 export const spinner = () => {
+	// essentially a mock
+	if (process.env.VITEST) {
+		const stub = (msg: string) => {
+			if (msg) updateStatus(`${leftT} ${msg}`);
+		};
+
+		return {
+			start: stub,
+			update: stub,
+			stop: stub,
+		};
+	}
+
```

Comment:
Ideally this would be part of the test setup outside of the main source code.
Could we add something to the tests that mocks out `spinner.start,update,stop`?
Perhaps that would require `spinner` to become a class so that it is easier to mock `Spinner.prototype.start()` etc.

Reply:
i've removed this and just mocked the `log-update` package in unit-tests. Can't mock the e2e tests as they run the built+bundled cli.js file – if we ever see output in e2e tests we'll have to make do with repeated lines from the spinner's setInterval